<launch>
  <node pkg="control" type="PIDService1.py" name="pid_service1" output="screen" />
  <node pkg="control" type="PIDService2.py" name="pid_service2" output="screen" /> 


  <node
    pkg="control"
    type="motor_tuning.py"
    name="motor_tuning"
    output="screen"
  />
  <node
    pkg="motor"
    type="motor_read_write.py"
    name="read_write_py_node"
    output="screen"
  /> 
</launch>